packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -624,26 +621,6 @@ const MainScreen = (props: MainScreenProps): JSX.Element => {
       break;
   }
 
-  /* @conditional-compile-remove(dtmf-dialer) */
-  if (dialpadScreen) {
-    pageElement = (
-      <>
-        <DtmfDialpadPage
-          mobileView={props.mobileView}
-          modalLayerHostId={props.modalLayerHostId}
-          options={props.options}
-          updateSidePaneRenderer={setSidePaneRenderer}
-          mobileChatTabHeader={props.mobileChatTabHeader}
-          latestErrors={latestErrors}
-          onDismissError={onDismissError}
-          /* @conditional-compile-remove(capabilities) */
-          capabilitiesChangedNotificationBarProps={capabilitiesChangedNotificationBarProps}
-          onSetDialpadPage={() => setDialpadScreen(!dialpadScreen)}
-        />
-      </>
-    );
-  }
-
```

Comment:
Ah  see. We are moving the dialpad page to the CallArrangement so that we can still see the control bar.

Reply:
better for performance since we are just changing the gallery content instead of page switching in the whole composite. (makes the new screens code simpler as well)